testing/integration/Cargo.toml

```diff
@@ -10,11 +10,10 @@ repository = "https://github.com/filecoin-project/ref-fvm"
 [dependencies]
 fvm = { version = "3.1.0", path = "../../fvm", default-features = false, features = ["testing"] }
 fvm_shared = { version = "3.1.0", path = "../../shared", features = ["testing"] }
-fvm_ipld_hamt = { version = "0.6.1", path = "../../ipld/hamt" }
-fvm_ipld_amt = { version = "0.5.1", path = "../../ipld/amt" }
 fvm_ipld_car = { version = "0.6.0", path = "../../ipld/car" }
 fvm_ipld_blockstore = { version = "0.1.1", path = "../../ipld/blockstore" }
 fvm_ipld_encoding = { version = "0.3.3", path = "../../ipld/encoding" }
+actors-v10 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", branch = "next", features = ["m2-native"] }
```

Comment:
We can't have this in the dependencies section as it'll prevent us from publishing to crates.io.